components/clp-py-utils/clp_py_utils/clp_config.py

```
@@ -1025,6 +1126,24 @@ def validate_log_ingestor_config(self):
             raise ValueError(msg)
         return self
 
+    @model_validator(mode="after")
+    def validate_compression_orchestration_config(self):
+        if CompressionOrchestration.SPIDER != self.package.scheduler:
```

Ruff (0.16.2)

[warning] Yoda condition detected

(SIM300)